lib/membrane_rtc_engine/engine.ex

```diff
@@ -223,6 +223,7 @@ defmodule Membrane.RTC.Engine do
   * `trace_ctx` is used by OpenTelemetry. All traces from this engine will be attached to this context.
   Example function from which you can get Otel Context is `get_current/0` from `OpenTelemetry.Ctx`.
   * `display_manager?` - set to `true` if you want to limit number of tracks sent from `#{inspect(__MODULE__)}.Endpoint.WebRTC` to a browser.
+  * `toilet_capacity` - declare capacity of buffer between endpoints and tee. If not provided it will be set to 200.
```

Comment:
This exposes a little bit internal engine architecture as someone must know what is tee. I would write it in more general way with a small suggestion when to use this option e.g.

```suggestion
  * `toilet_capacity` - sets capacity of buffer between engine and endpoints. Use it when you expect bursts of data for your tracks. If not provided it will be set to 200.
  ```